### PR TITLE
[Consignments] Welcome screen for the consignment flow

### PR DESCRIPTION
**Pull request by orta**
screen shot 2017-05-18 at 15 58 10

This would make a really nice SVG test 👍. 

Skip New Tests

### CODE REVIEW

File: src/lib/components/buttons/flat_white.tsx [hunk 1]
```diff
@@ -77,6 +77,7 @@ const styles = StyleSheet.create({
     alignItems: "center",
     borderColor: colors["gray-regular"],
     borderWidth: 1,
+    flex: 1,
```
Comment (orta) [inline src/lib/components/buttons/flat_white.tsx]: Nope

screen shot 2017-05-18 at 16 10 46

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Missing Test Files:

- `src/lib/components/buttons/__tests__/flat_white-tests.tsx`
- `src/lib/components/consignments/components/__tests__/circle_image-tests.tsx`
- `src/lib/components/consignments/images/__tests__/email@2x.png`
- `src/lib/components/consignments/images/__tests__/hammer@2x.png`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Could not find stories using these components:

- `WhiteButton`

</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by orta** [comment]
Heh, looks like the check for tests needs to ensure it's a TypeScript file 💯

File: tsconfig.json [hunk 2]
```diff
@@ -13,7 +13,8 @@
     "include": [
         "typings/index.d.ts",
         "src/**/*.ts",
-        "src/**/*.tsx"
+        "src/**/*.tsx",
+        "dangerfile.ts"
     ],
     "exclude": [
         "dangerfile.ts",
```
Comment (orta) [inline tsconfig.json]: I discovered this trick when building the yeoman generator. The Dangerfile can be included by the TSServer (dev-time) and ignored by TSC (build-time).

**Comment by orta** [comment]
Would recommend not merging, but approving/requesting changes. There's three PRs lined up and this merges into #548

File: src/lib/components/consignments/setup/welcome.tsx [hunk 3]
```diff
@@ -1,39 +1,51 @@
 import * as React from "react"
-import * as Relay from "react-relay"
-import Button from "../../buttons/inverted_button"
-import Info from "./info"
+
+import Button from "../../buttons/flat_white"
```
Comment (mennenia) [inline src/lib/components/consignments/setup/welcome.tsx]: ☕️ so hipster

File: src/lib/components/buttons/flat_white.tsx (hunk 1 above)
Comment (orta) [inline src/lib/components/buttons/flat_white.tsx]: Yeah - I went back and checked after, turned out it looked bad - it's what made me decide it was time to start https://github.com/orta/jest-snapshots-svg

**Comment by orta** [comment]
Just ran svg-snapshots against this: 

screen shot 2017-05-22 at 10 32 21

**Comment by orta** [comment]
Once I disabled the black BG anyway

**Comment by orta** [comment]
OK, this is good - I got the testing pretty far, might merge it in once I've got all the other PRs ready.

**Comment by mennenia** [comment]
🎉

**Pull request merged**